packages/core/less/verticalnavdivider.less

```diff
@@ -0,0 +1,13 @@
+.vertical-sub-header-pf {
+  line-height: 25px;
+  font-size: @font-size-base;
+  margin-left: 20px;
+  padding: 5px 0 2px;
+  font-weight: bold;
+  color: @nav-pf-vertical-secondary-item-color;
```

Comment:
I realize that this is the variable I recommended, but after seeing it and chatting with @mcarrano, can we use this one instead: `@nav-pf-vertical-secondary-color`? This will help make the text stand out a bit more than it does now.